backend/molgenis-emx2/src/main/java/org/molgenis/emx2/User.java

```diff
@@ -0,0 +1,13 @@
+package org.molgenis.emx2;
+
+public class User {
```

Comment:
what about a user without a usename ?

Reply:
at the moment all users HAVE to have user name. Later on, with oicd, we can revisit this.

Reply:
Added User(String username) constructor and assert username != null